first session/assignment/shoppingmall.html

```diff
@@ -0,0 +1,191 @@
+                    <img src="./static/img/허리보호대.jpg" alt="허리보호대">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>허리보호대</h3>
+                    <p>가격: 6,000원</p>
+                </div>
+                <div class="product">
+                    <img src="./static/img/텐트.jpg" alt="텐트">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>텐트</h3>
+                    <p>가격: 100,000원</p>
+                </div>
+        
+                <div class="product">
+                    <img src="./static/img/책.jpg" alt="책">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>책</h3>
+                    <p>가격: 10,000원</p>
+                </div>
+        
+                <div class="product">
+                    <img src="./static/img/인형.jpg" alt="인형">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>텐트</h3>
+                    <p>가격: 50,000원</p>
+                </div>
+                <div class="product">
+                    <img src="./static/img/모자.jpg" alt="모자">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>모자</h3>
+                    <p>가격: 30,000원</p>
+                </div>
+
+                <div class="product">
+                    <img src="./static/img/청바지.jpg" alt="청바지">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>모자</h3>
+                    <p>가격: 20,000원</p>
+                </div>
+                <!--cla-->
+                <div class="product">
+                    <img src="./static/img/th.jpg" alt="치마">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>치마</h3>
+                    <p>가격: 20,000원</p>
+                </div>
+
+            </div>
+            <!-- 상품 리스트 -->
+        </section>
+        <div>
+            <h2>운동 용품</h2>
+            <section id="category1">
+                <div class="product">
+                    <img src="./static/img/자전거.jpg" alt="자전거">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>자전거</h3>
+                    <p>가격: 10,000원</p>
+                </div>
+    
+                <div class="product">
+                    <img src="./static/img/아령.jpg" alt="아령">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>아령</h3>
+                    <p>가격: 5,000원</p>
+                </div>
+                <div class="product">
+                    <img src="./static/img/허리보호대.jpg" alt="허리보호대">
+                    <div><button class="like-button">♥</button></div>
+                    <h3>허리보호대</h3>
+                    <p>가격: 6,000원</p>
+                </div>
+               
+                <!-- 추
+                <!-- 상품 리스트 -->
+            </section>
+        </div>
```

Comment:
지금 section마다 바깥에 감싸는 div 태그가 존재해요!! 그런데 이 div태그가 사실 아무 역할도 하고 있지 않아요!! 아무 역할을 하지 않는 div는 사용하지 않는 것이 좋아요!!

```suggestion
        <section id="category1">
              <h2>운동 용품</h2>
              <ul class="itemlist어쩌구저쩌구">
                <li class="product">
                    <img src="./static/img/자전거.jpg" alt="자전거">
                    <button class="like-button">♥</button>
                    <h3>자전거</h3>
                    <p>가격: 10,000원</p>
                </li>
    
                <li class="product">
                    <img src="./static/img/아령.jpg" alt="아령">
                    <button class="like-button">♥</button>
                    <h3>아령</h3>
                    <p>가격: 5,000원</p>
                </li>
                <li class="product">
                    <img src="./static/img/허리보호대.jpg" alt="허리보호대">
                    <button class="like-button">♥</button>
                    <h3>허리보호대</h3>
                    <p>가격: 6,000원</p>
                </li>
             </ul>
        </section>
```

불필요한 태그들을 제거하고 시맨틱하게 태그들을 수정해봤는데 참고해서 다른 section들도 수정해보면 많은 도움이 될 거예요 :)